### ISSUE TITLE
Add Langton's Ant

### ISSUE THREAD

**Issue by the-ethan-hunt** [OpenGenus/cosmos#1292]
Like Conway's game of life, Langton's ant is another cellular automation. Check out [this](https://en.wikipedia.org/wiki/Langton%27s_ant) for more details

**Comment by fisian** [comment]
If there are more implementations of cellular automata it would be nice if they would not be categorized in the 'uncategorized' section.

**Comment by AdiChat** [comment]
Yes @fisian We can create a new root category named `cellular_automata` 👍

**Comment by ron-debajyoti** [comment]
I can add Langton's Ant in python using turtle package
